splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/DecfloatIT.java

```diff
@@ -64,15 +63,25 @@
     @Rule
     public SpliceWatcher methodWatcher = new SpliceWatcher(CLASS_NAME);
 
+    @Before
+    public void setUp() throws Exception{
+        SpliceWatcher.ConnectionBuilder connBuilder = methodWatcher.connectionBuilder();
+        if (useSpark != null){
+            connBuilder.useOLAP(useSpark);
+        }
+        conn = connBuilder.build();
+        conn.setAutoCommit(false);
+    }
+
```

Comment:
hm maybe you could have extended methodWatcher to support sth like methodWatcher.setConnection . I mean this is not a big thing, but i'm wondering if we need TestConnection AND SpliceWatcher, which have a lot of overlap in functionality. (Not a big thing since it's only tests)